boards/PDM/Src/Io/Io_Efuse_22XS4200.c

```diff
@@ -0,0 +1,198 @@
+#include "Io_Efuse_22XS4200.h"
+
+// The SPI handle for the SPI device the E-Fuses are connected to
+static SPI_HandleTypeDef *efuse_spi_handle;
+
+void Io_Efuse_22XS4200_Init(SPI_HandleTypeDef *const hspi)
+{
+    efuse_spi_handle = hspi;
+}
+
+void Io_Efuse_22XS4200_configureEfuse(struct E_Fuse *e_fuse)
```

Comment:
Shouldn't we also configure e-fuse in `main()`? Would it make sense to combine `Init` with `configureEfuse`?

Reply:
Ya this should have be called in main(), I forgot to add the call. Hmm in the App layer I have a function that call's:
`void Io_Efuse_22XS4200_configureEfuse(struct E_Fuse *e_fuse)`
On each of the Efuse's and configures them all in one call.
Should I make a init function in the App layer also
```
void App_Efuses_Init(SPI_HandleTypeDef *const hspi)
{
    ....
}
```
And just call that in main():
`App_Efuses_Init(&hspi2);`

Reply:
No the App layer should have no information about the SPI handle, `Io_Efuse_22XS4200_configureEfuse` should be done entirely in the IO layer